src/effects/native/balanceeffect.cpp

```diff
@@ -0,0 +1,176 @@
+#include "balanceeffect.h"
+
+#include "util/defs.h"
+
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
22000 Hz is the maximum we can express with 44100 Hz sample rate and already above the 20 kHz maximum for a HiFi signal. So we are almost fully dry if we turn the knob fully clock wise. 
We are not entirely fully dry, because of the phase shift of a Linkwitz Rily filter, but that should be unhearable.